src/wayland/protocols.c

```diff
@@ -0,0 +1,13 @@
+#include "protocols/xdg-shell.h"
+#include "protocols/wlr-layer-shell-unstable-v1.h"
+#include "protocols/wlr-foreign-toplevel-management-unstable-v1.h"
+#include "protocols/idle.h"
+
+#ifdef HAVE_WL_CURSOR_SHAPE
+#include "protocols/cursor-shape-v1.h"
+#include "protocols/tablet-unstable-v2.h"
+#endif
+
+#ifdef HAVE_WL_EXT_IDLE_NOTIFY
+#include "protocols/ext-idle-notify-v1.h"
+#endif
```

Comment:
Is this file used anywhere? It might be used in wl.c instead of inporting the protocols separately.